src/views/Upload.vue

```diff
@@ -36,7 +36,26 @@
 					@text-change="updateText"
 				/>
 			</div>
-			<div class="flex flex-row justify-end items-end text-center w-full px-2 md:px-20 md:text-right pb-2 md:pb-10 mt-10 md:mt-10 ">
+
+			<div class="flex flex-row justify-end items-end text-center w-full px-2 md:px-20 md:text-right pb-2 md:pb-10 mt-10 md:mt-10 ">		
+				<div class="flex flex-row">	
+					<input 
+						v-model="checked" 
+						type="checkbox" 
+						class="m-2"
+					>
+					<div v-if="checked == true" class="flex-row pr-5">
+						<input 
+							v-model="message" 
+							class="border border-dark" 
+							placeholder="mail@mail.sk"
+						>
```

Comment:
What if we displayed the input all the time, but it would be disabled / enabled according to the checkbox?